SoftLayer/CLI/image/import.py

```diff
@@ -18,13 +18,38 @@
 @click.option('--os-code',
               default="",
               help="The referenceCode of the operating system software"
-                   " description for the imported VHD")
+                   " description for the imported VHD, ISO, or RAW image")
+@click.option('--ibm-api-key',
+              default="",
+              help="The IBM Cloud API Key with access to IBM Cloud Object "
+                   "Storage instance.")
+@click.option('--root-key-id',
+              default="",
+              help="ID of the root key in Key Protect")
+@click.option('--wrapped-dek',
+              default="",
+              help="Wrapped Decryption Key provided by IBM KeyProtect")
+@click.option('--kp-id',
+              default="",
+              help="ID of the IBM Key Protect Instance")
+@click.option('--cloud-init',
```

Comment:
Click has an option `is_flag=True,` which should be used here. This way you don't need the `default=""` bit

Reply:
Will change in the next commit.